docs/markdown/ios/card/subcomponents/card.pins.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Card.Pins
+description: The Boards preview block or “Boards PB” utilizes two of the Board and Section components to present a collection of imagery that typically represents a Card.Boards.
+fullwidth: true
+---
+
+<ImgContainer src="https://github.com/pinterest/gestalt/assets/96082362/ec0692aa-cc1a-42df-80c9-f0e212100fe4" alt="Example of a Card container with a Card.Boards preview that contains two Card.Boards representations." noPadding color="background-elevation-accent"/>
+
+
+## Usage guidelines
+
+<TwoCol>
+<Group>
+<Do title="When to use"/>
+- To present multiple entry points into a single destination containing Pins (ex: stream)
+- When the user goal is best supported by focusing them on one Pin at a time (ex: watching videos in a related stream, evaluating Pins with a high intent mindset)
+
+</Group>
+<Group>
+<Dont title="When not to use" />
+- When each tap target leads to a different destination (ex: different streams)
+- When the goal is to direct Pinners to a landing experience that displays more than one Pin (e.g. a topical Pin feed). 
+</Group>
+</TwoCol>
+
+## Best practices
+<TwoCol>
+<Group>
+<Do title="Do"/>
```

Comment:
These don't flow very well — we typically treat Do/Don't's as sentences where "Do" or "Don't" is the first word of the sentence. "Do Pin previews can flex…" is nonsensical.

Reply:
agree! i'll see if I can rewrite it, but this may have to get updated later.